scripts/netlify.sh

```diff
@@ -15,9 +15,6 @@ yarn examples:router:build
 yarn examples:media:build
 yarn examples:storybook:build
 
-# build dev-novel
-MODE=build webpack --config webpack.demo.js
```

Comment:
What is this replaced with?

Reply:
yarn examples:storybook:build

Reply:
Ah yeah, I didn't see the change was already done!